GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -53,7 +55,15 @@ public InProcessMountVerb()
             Default = "false",
             Required = false,
             HelpText = "Service initiated mount.")]
-        public string StartedByService { get; set; }
+        public string StartedByService { get; set; }
+
+        [Option(
+            'b',
```

Comment:
Why `b`?  Background?  Looks like we can't use `v` for Verb but would `i` (initiated) or `p` (prepare) be better?

Reply:
Yep `b` was for background.  

Since these options are for `GVFS.Mount` itself and not `gvfs <verb>` no one should ever be calling them directly.  I'm tempted to leave this as-is since chances are no one will ever manually set this, but please let me know if you'd rather I change it.

Reply:
Nope `b` works for me.